styled remaining admin pages

eric-ls opened the pull request.

eric-ls committed:
styled remaining admin pages

eric-ls committed:
styled remaining admin pages

nnarayen commented:
HOLY SHIT this is amazing! I have no logic comments, just a couple questions about styling:

1) Would it be possible to reduce the space in between ApplicantCards on the Breakdown page? Currently it looks like

![screen shot 2016-01-20 at 1 13 20 am](https://cloud.githubusercontent.com/assets/1946683/12444350/53d4da3e-bf13-11e5-89e1-79ac1ec6a077.png)

But I think my old version without the space (and slightly thicker borders) might look slightly cleaner:

![screen shot 2016-01-20 at 1 15 57 am](https://cloud.githubusercontent.com/assets/1946683/12444356/6378a4e8-bf13-11e5-8fae-688999bdc15f.png)

What are your thoughts?

2) Can you make the textarea on the Compose Emails section bigger? They might be a couple paragraphs long, so it'll be nice to see the entire email

3) On the Schedule Interviews page, when you click on an applicant name it should redirect to their overview page. Can you make the cursor pointer and change the background color on hover slightly?

But seriously overall this is ridiculous, you are web styl1ng g0d

nnarayen commented on app/assets/javascripts/components/applicants/admin_applicant_info.jsx:
```diff
@@ -5,26 +5,30 @@ class AdminApplicantInfo extends React.Component {
-                { this._renderInput("phone", "Phone Number") }
+                <div className="col-md-7 col-sm-7">
+                    <h2 className="applicant-name">
+                        {this.props.applicant["full_name"]}
```
should be able to access with this.props.applicant.full_name

eric-ls committed:
addressed PR comments

eric-ls committed:
addressed PR comments

Commit:
Implementing password reset functionality

Summary:
Various refactors, new ability to reset passwords. Added LetterOpener
gem

Test Plan: Route to home and attempt to reset password

eric-ls committed:
additional changes

eric-ls committed:
rebased
